v3/lints/cabf_cs_br/lint_cs_crl_distribution_points.go

```diff
@@ -0,0 +1,62 @@
+package cabf_cs_br
+
+import (
+	"strings"
+
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+)
+
+/*7.1.2.3 b. cRLDistributionPoints
+This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the
+HTTP URL of the CA’s CRL service*/
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_cs_crl_distribution_points",
+			Description:   "This extension MUST be present. It MUST NOT be marked critical. It MUST contain the HTTP URL of the CA's CRL service",
+			Citation:      "CABF CS BRs 7.1.2.3.b",
+			Source:        lint.CABFCSBaselineRequirements,
+			EffectiveDate: util.CABF_CS_BRs_1_2_Date,
+		},
+		Lint: NewCrlDistributionPoints,
+	})
+}
+
+type crlDistributionPoints struct{}
+
+func NewCrlDistributionPoints() lint.LintInterface {
+	return &crlDistributionPoints{}
+}
+
+func (l *crlDistributionPoints) CheckApplies(c *x509.Certificate) bool {
+	return util.IsCodeSigning(c.PolicyIdentifiers) && util.IsSubscriberCert(c)
```

Comment:
This indeed does complex, fast, but that's the nature of the beast! :smile: 

Thank you for summarizing this for me and updating `base.go` with the blanket check. I've given some code suggestions in the `CheckApplies` procedures in these lints. Beyond this specific conversation, I think that these lints are good to go!

```suggestion
	return util.IsSubscriberCert(c) || util.IsSubCA(c)
```